icons.yml

```diff
@@ -7,390 +7,327 @@ mappings:
-    color: '#FFC324'
+    color: '#F3C600'
   - subclass: fast_food
     iconName: fast-food-11
-    color: '#FF8D2B'
-  - subclass: dry_cleaning
-    iconName: laundry-11
-    color: '#8A9CF6'
-  - subclass: laundry
-    iconName: laundry-11
-    color: '#8A9CF6'
+    color: '#EF8000'
   - subclass: bus_stop
     iconName: bus-11
-    color: '#5CA6FB'
+    color: '#8665E7'
   - subclass: subway
     iconName: rail-metro-11
-    color: '#5CA6FB'
+    color: '#8665E7'
   - subclass: ferry_terminal
     iconName: ferry-11
-    color: '#5CA6FB'
+    color: '#8665E7'
   - subclass: entrance
     iconName: entrance-11
-    color: '#5CA6FB'
+    color: '#8665E7'
   - subclass: park
     iconName: park-alt1-11
-    color: '#00CB4F'
+    color: '#76BC54'
   - subclass: dog_park
     iconName: dog-park-11
-    color: '#00CB4F'
+    color: '#76BC54'
   - subclass: playground
     iconName: playground-11
-    color: '#00CB4F'
+    color: '#5C6F84'
   - subclass: food_court
-    iconName: picnic-site-11
-    color: '#00CB4F'
+    iconName: food-court-11
+    color: '#5C6F84'
   - subclass: picnic_site
     iconName: picnic-site-11
-    color: '#00CB4F'
+    color: '#76BC54'
   - subclass: garden
     iconName: garden-11
-    color: '#00CB4F'
-  - subclass: caravan_site
-    iconName: campsite-11
-    color: '#00CB4F'
-  - subclass: camp_site
-    iconName: campsite-11
-    color: '#00CB4F'
+    color: '#76BC54'
   - subclass: veterinary
     iconName: veterinary-11
-    color: '#8A9CF6'
+    color: '#4AA0E7'
   - subclass: zoo
     iconName: zoo-11
-    color: '#00CB4F'
+    color: '#E93865'
   - subclass: aquarium
     iconName: aquarium-11
-    color: '#8A9CF6'
+    color: '#E93865'
   - subclass: gallery
     iconName: art-gallery-11
-    color: '#8A9CF6'
-  - subclass: post_box
-    iconName: post-11
-    color: '#8A9CF6'
-  - subclass: school
-    iconName: school-11
-    color: '#A3A3A3'
-  - subclass: cemetery
-    iconName: cemetery-11
-    color: '#00CB4F'
-  - subclass: grave_yard
-    iconName: cemetery-11
-    color: '#00CB4F'
-  - subclass: hospital
-    iconName: hospital-11
-    color: '#FF5E6A'
+    color: '#E93865'
   - subclass: doctors
     iconName: doctor-11
-    color: '#FF5E6A'
+    color: '#4AA0E7'
   - subclass: clinic
     iconName: doctor-11
-    color: '#FF5E6A'
+    color: '#4AA0E7'
   - subclass: dentist
     iconName: dentist-11
-    color: '#FF5E6A'
-  - subclass: kindergarten
-    iconName: school-11
-    color: '#A3A3A3'
+    color: '#4AA0E7'
   - subclass: museum
     iconName: museum-11
-    color: '#8A9CF6'
-  - subclass: post_office
-    iconName: post-11
-    color: '#8A9CF6'
+    color: '#E93865'
   - subclass: public_building
     iconName: building-15
-    color: '#000000'
-  - subclass: attraction
-    iconName: attraction-11
-    color: '#8A9CF6'
-  - subclass: viewpoint
-    iconName: attraction-11
-    color: '#8A9CF6'
+    color: '#495063'
   - subclass: artwork
     iconName: attraction-11
```

Comment:
artwork are almost street furniture, and they are really too ugly in pink : can we have them in Unclassified colour instead of Entertainment one ?
![screenshot_2018-11-08 luxembourg](https://user-images.githubusercontent.com/919962/48212210-69c80980-e37b-11e8-970e-274d7ec5b62e.png)

Reply:
Should I change artwork only or artwork + attraction ? 

Example with only artwork
![artwork only](https://user-images.githubusercontent.com/5153882/48268236-b9b6d700-e434-11e8-951e-374c01df6a5e.png)